daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1528,6 +1528,10 @@ message DefInterface {
   int32 tycon_interned_dname = 2;
   repeated InterfaceChoice choices = 3;
   repeated InterfaceMethod methods = 4;
```

Comment:
Suggestion: 
We could called those field `virtualChoices` and `virtualMethods`.

Reply:
I'll leave renaming these to a later PR.